src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (currentValue, num) {
+      return currentValue + num;
+    },
```

Comment:
1. simplify a record
2. use `this`
```suggestion
    add(x) {
      this.result += x;

      return this;
    },
```